dht_test.go

```diff
@@ -1414,8 +1414,10 @@ func testFindPeerQuery(t *testing.T,
 	waitForWellFormedTables(t, dhts, minRTRefreshThreshold, minRTRefreshThreshold, 5*time.Second)
 
 	for _, d := range dhts {
-		if err := <-d.RefreshRoutingTable(); err != nil {
-			t.Fatal(err)
+		if len(d.RoutingTable().ListPeers()) > 0 {
+			if err := <-d.RefreshRoutingTable(); err != nil {
+				t.Fatal(err)
+			}
```

Comment:
The test had to be modified because of https://github.com/libp2p/go-libp2p-kad-dht/pull/549#issuecomment-609550787